tests/components/flux_led/test_config_flow.py

```diff
@@ -0,0 +1,463 @@
+            CONF_DEVICES: {
+                "1_1_1_1": {
+                    CONF_NAME: "TestLight",
+                    CONF_HOST: "1.1.1.1",
+                }
+            },
+        },
+    ).add_to_hass(hass)
+
+    await setup.async_setup_component(hass, "persistent_notification", {})
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN,
+        context={"source": config_entries.SOURCE_IMPORT},
+        data={
+            CONF_AUTOMATIC_ADD: False,
+            CONF_DEVICES: {
+                "1_1_1_1": {
+                    CONF_NAME: "TestLight",
+                    CONF_HOST: "1.1.1.1",
+                }
+            },
+        },
+    )
+    assert result["type"] == "abort"
+    assert result["reason"] == "single_instance_allowed"
+
+
+async def test_options_set_global_options(hass):
+    """Test set global options through options flow."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+
+    entry = MockConfigEntry(
+        domain="flux_led",
+        data={
+            CONF_AUTOMATIC_ADD: False,
+            CONF_EFFECT_SPEED: DEFAULT_EFFECT_SPEED,
+            CONF_DEVICES: {
+                "1_1_1_1": {
+                    CONF_NAME: "TestLight",
+                    CONF_HOST: "1.1.1.1",
+                }
+            },
+        },
+    )
+    entry.add_to_hass(hass)
+
+    with patch(
+        "homeassistant.components.flux_led.light.BulbScanner.scan",
+        return_value=[
+            {
+                "ipaddr": "1.1.1.1",
+                "id": "test_id",
+                "model": "test_model",
+            }
+        ],
+    ):
+        assert await hass.config_entries.async_setup(entry.entry_id)
+        await hass.async_block_till_done()
+
+    result = await hass.config_entries.options.async_init(entry.entry_id)
+
+    assert result["type"] == "form"
+    assert result["step_id"] == "prompt_options"
+
+    result2 = await hass.config_entries.options.async_configure(
+        result["flow_id"],
+        user_input={CONF_AUTOMATIC_ADD: True, CONF_EFFECT_SPEED: 80},
+    )
+
+    assert result2["type"] == "create_entry"
+    assert result2["data"] == {
+        "global": {
+            CONF_AUTOMATIC_ADD: True,
+            CONF_EFFECT_SPEED: 80,
+        }
+    }
+
+
+async def test_options_add_and_remove_new_light(hass):
+    """Test manual adding and removing of new light through options flow."""
+    await setup.async_setup_component(hass, "persistent_notification", {})
+
+    entry = MockConfigEntry(
+        domain="flux_led",
+        data={
+            CONF_AUTOMATIC_ADD: False,
+            CONF_EFFECT_SPEED: DEFAULT_EFFECT_SPEED,
+            CONF_DEVICES: {
+                "1_1_1_1": {
+                    CONF_NAME: "TestLight",
+                    CONF_HOST: "1.1.1.1",
+                }
+            },
+        },
+    )
+    entry.add_to_hass(hass)
+
+    result = await hass.config_entries.options.async_init(entry.entry_id)
```

Comment:
We need to set up the integration in all the options flow tests.

Reply:
Done... sorry!